### DISCUSSION TITLE
Sample usage?

### DISCUSSION THREAD

**Discussion by lbdyck** [oSoWoSo/gumak#1] [Q&A]
I would love to see a sample use of this. It sounds promising.

**Comment by zen0bit** [comment]
Now just lets you quickly choose gum commands and generate without typing.  Results are stored in temp files(each in its own).

(Now will be rewriten if you run it for second time)

From there user can copy it to own script.

Anything could change, Just started prototype...

**Comment by lbdyck** [comment]
i look forward to following your progress

**Comment by zen0bit** [comment]
I am using it now for quickly generate gum commands for some coretools etc. Will add them later (after more polishing)

I wanna crate TUIs for most of terminal base tools which I use daily. Already working on. gme is first example... More will come

Let's make terminal apps more Glamorous 🐧

**Comment by lbdyck** [comment]
This is my 1st attempt at a generalized TUI using `gum` - feel free to use/share/enhance - i'm just learning shell scripting and gum

```
#!/bin/sh
/bin/clear
gum style \
        --foreground 212 --border-foreground 212 --border double \
        --align center --width 50 --margin "1 2" --padding "2 4" \
        'z/OS Open Tools Man Viewer'

if [ "$#" -eq 0 ]; then
    ARG=$(gum input --prompt "Enter a command for a manpage or a word to search for:" \
    --placeholder="e.g. ssh" --value=" ")
else
    ARG=$1
fi

export GUM_INPUT_CURSOR_FOREGROUND="#FF0"
export GUM_INPUT_PROMPT_FOREGROUND="#0FF"

echo "Select the man page to view:"
MAN=$(man -k $ARG | cut -d" " -f1,3- | tail -n +2 | gum choose)
MAN=$(echo $MAN | cut -d" " -f1)
man $MAN | gum pager
```

**Reply by zen0bit** [reply]
Added video to readme
